### PR TITLE
Add datadog

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: modules/kubernetes/datadog/templates/values.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+datadog:
+  apiKey: ${api_key}
+  clusterName: ${location}-${environment}
+  site: datadoghq.eu
```
Comment (simongottschlag) [inline modules/kubernetes/datadog/templates/values.yaml.tpl:4]: Make site a variable that defaults to datadoghq.eu

File: modules/kubernetes/datadog/templates/values.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    enabled: true
+    containerCollectAll: true
+    containerCollectUsingFiles: true
+  apm:
+    enabled: true
+  systemProbe:
+    enabled: false
+  env:
+    - name: DD_KUBELET_TLS_VERIFY
+      value: "false"
```
Comment (simongottschlag) [inline modules/kubernetes/datadog/templates/values.yaml.tpl:23]: Is this one still required?

**Review by simongottschlag — COMMENTED** [review]
Two comments

File: modules/kubernetes/datadog/templates/values.yaml.tpl (hunk 2 above)
Comment (phillebaba) [reply modules/kubernetes/datadog/templates/values.yaml.tpl:23]: no that should now be removed

File: modules/kubernetes/datadog/templates/values.yaml.tpl (hunk 1 above)
Comment (phillebaba) [reply modules/kubernetes/datadog/templates/values.yaml.tpl:4]: sounds good

File: modules/kubernetes/datadog/templates/values.yaml.tpl [hunk 3]
```diff
@@ -0,0 +1,31 @@
+datadog:
+  apiKey: ${api_key}
+  clusterName: ${location}-${environment}
+  site: ${datadog_site}datadoghq.eu
```
Comment (simongottschlag) [inline modules/kubernetes/datadog/templates/values.yaml.tpl:4]: This seems wrong

**Review by simongottschlag — APPROVED** [review]
LGTM

**Pull request merged**